### PR TITLE
Create file structure

### LINKED ISSUE #1: [html, css, js] Create file structure

### PR DESCRIPTION
**Pull request by Bas-Shiekh**

### CODE REVIEW

File: html/question.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/question.css">
```
Comment (mwadia) [inline html/question.html]: write the correct path.

**Review by fadezak100 — CHANGES_REQUESTED** [review]
Add assets folder for images and other visual assets

**Review by mwadia — CHANGES_REQUESTED** [review]
pls, see the comment.

**Review by mwadia — APPROVED** [review]
Great work basel ya batal!

**Review by fadezak100 — APPROVED** [review]
LGTM

**Pull request merged**